gravitee-am-ui/src/app/services/provider.service.ts

```diff
@@ -63,7 +63,6 @@ export class ProviderService {
       mappers: provider.mappers,
       roleMapper: provider.roleMapper,
       groupMapper: provider.groupMapper,
-      passwordPolicy: provider.passwordPolicy,
```

Comment:
question: why did you remove this line? I'm afraid of a regression with the PasswordPolicy feature linked to the IDP if we commit this change.